portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -66,6 +66,16 @@ definition {
 		Click(locator1 = "FormFields#OPTIONS_DELETE_ROW_BUTTON");
 	}
 
+	macro deleteSpecificOptionFromMultipleSelection {
+		Click(
+			key_value = "${key_valueOption}",
+			locator1 = "FormFields#DELETE_OPTION_FROM_MULTIPLE_SELECTION");
+
+		AssertElementNotPresent(
+			key_value = "${key_valueOption}",
```

Comment:
@piresdm, please rename `key_option` to `key_valueOption` to keep naming consistent. :wink:

Reply:
@natocesarrego done!